### ISSUE TITLE
google cal widget broken?

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-web#13261]

**Comment by fbruetting** [comment]
Every time I click on the bar, it opens a scalar.vector.im website, showing me “Sorry, this calendar can not be displayed” with an image placeholder (as the content cannot be loaded). When I then go back to Riot, the widget opened up, but shows the same. Worked fine before for months. 🤔

**Comment by turt2live** [comment]
A long time ago Google changed it so only public calendars can be embedded for some reason. It seems like I'm the only one failing to reproduce this issue for the last 4 months though :|

**Comment by fbruetting** [comment]
For me it’s a public calendar, it’s also embedded on our webpage.

**Comment by fbruetting** [comment]
I checked and it’s a public calendar which also is embedded in our webpage.

**Comment by t3chguy** [comment]
Scalar 502 Bad gateway with a public gcal

![image](https://user-images.githubusercontent.com/2403652/79699246-3256cf80-8286-11ea-8efd-5f6b63dff10c.png)

**Comment by fbruetting** [comment]
Additionally, I find it weird that clicking on the minimized bar not only opens the widget, but also opens a new webpage in the Browser.

**Comment by hlesueur** [comment]
Hello everyone, I tried to implement google agenda. 
==> It failed in the ggl agenda widget, no matter which settings I have for my calendar (even if it's public, I reproduced with 2 different agendas, on firefox and chrome and desktop client). 
(I get a banana 🙂 )
That may be a bug report.

However my request is more about the method I almost got to work:
I use the custom widget and I can make it work with an url like 
https://calendar.google.com/calendar/embed?src=id@group.calendar.google.com
either IF the calendar is public, OR it's private and I'm logged with my ggl account (in firefox and chrome, did not try in desktop client)

I'd like to avoid making the agenda public or/and being logged to ggl (all the members of my community don't have a ggl account, yet I'd like them to see the agenda)

==> So I'd like to put the **private** agenda url into the custom widget 
Private url is given in the google agenda settings in its .ics format but not (no longer?) in the html format. 
This private link contains a token which can be revoked. This is a safe way to proceed I think.
The ics looks like 
https://calendar.google.com/calendar/ical/id@group.calendar.google.com/private-xxxbigtoken/basic.ics

My issue is: I could not find the proper syntax for the html url -the one I can put in the custom widget-
is there any keyword argument for the token? any specific path for a private agenda?

Conversely, would there be a way such that the (another) widget accepts .ics ?

Thanks a lot
